tox.ini

```diff
@@ -74,7 +74,8 @@ deps =
 commands =
 	python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
     pip-sync requirements/tests.txt requirements/pandapower.txt
-    pip install -e .
+	# https://stackoverflow.com/a/65147477/14830405
+    pip install -e . --use-deprecated=legacy-resolver
```

Comment:
The legacy resolver has been completely removed from pip since January 2021 and will not work with our pip version, since it's `> 21.0`.